crates/services/upgradable-executor/src/instance.rs

```diff
@@ -108,9 +167,23 @@ impl Instance {
 }
 
 impl<Stage> Instance<Stage> {
-    const LATEST_HOST_MODULE: &'static str = "host_v0";
+    const V0_HOST_MODULE: &'static str = "host_v0";
+    const LATEST_HOST_MODULE: &'static str = "host_v1";
```

Comment:
In general I would avoid using a `LATEST_HOST_MODULE` to refer to the latest module, as it is more error prone when introducing new versions. E.g. when we introduce V2, a new `add_method_v1` function will need to be added and the latest `peek_next_txs_size` will need to be refactored to be in `v1`.